examples/complete/provider.tf

```diff
@@ -0,0 +1,7 @@
+
+provider "aws" {
+  region = "us-east-2"
+  assume_role {
+    role_arn = "arn:aws:iam::128840427886:role/OrganizationAccountAccessRole"
+  }
+}
```

Comment:
Please move this block back to `main.tf` and remove the `role_arn` assumption

Reply:
sry, accidentally checked in my test setup ...